compiler/src/dotty/tools/dotc/semanticdb/ExtractSemanticDB.scala

```diff
@@ -79,6 +79,7 @@ class ExtractSemanticDB extends Phase:
       || sym.isLocalDummy
       || sym.is(Synthetic)
       || sym.isSetter
+      || sym.isTopLevelWrapperObject
```

Comment:
Why isn't the wrapper marked Synthetic?

Reply:
I got rid of those new tests for wrapper objects and added synthetic flag